activerecord/lib/active_record/database_configurations.rb

```diff
@@ -184,17 +184,13 @@ def walk_configs(env_name, config)
         end
       end
 
-      def resolve_symbol_connection(env_name, pool_name)
-        db_config = find_db_config(env_name)
-
-        if db_config
-          config = db_config.configuration_hash.dup
-          db_config = DatabaseConfigurations::HashConfig.new(db_config.env_name, db_config.name, config)
+      def resolve_symbol_connection(name, pool_name)
+        if db_config = find_db_config(name)
```

Comment:
How about naming `env` consistently with others? It is a bit hard to understand if `name` is `db_config.name` or `db_config.env_name`.

https://github.com/rails/rails/blob/1da6a90699da61e954267048c7382cc7ad2e384d/activerecord/lib/active_record/database_configurations.rb#L70-L71

Reply:
I think you mean to link to the `find_db_config` name and yea it is confusing. Sometimes we want env and sometimes we want name. In this case in resolve_symbol_connection we want name. I'm going to think about the other methods over the weekend and come up with a better naming for these.